src/Map/src/Twig/MapRuntime.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Twig;
+
+use Symfony\UX\Map\Map;
+use Symfony\UX\Map\Renderer\RendererInterface;
+use Twig\Extension\RuntimeExtensionInterface;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ *
+ * @internal
+ */
+final class MapRuntime implements RuntimeExtensionInterface
+{
+    public function __construct(
+        private readonly RendererInterface $renderer,
+    ) {
+    }
+
+    public function renderMap(Map $map, array $attributes = []): string
+    {
+        return $this->renderer->renderMap($map, $attributes);
+    }
+}
```

Comment:
You don't need this class, i think you can tag the RendererInterface with 'twig.runtime'

cf Icons in
- Icons/config/services.php 
- Icons/src/DependencyInjection/UXIconsExtension.php

Reply:
Mmmmh, I prefer having dedicated classes to do that, the combo `FooExtension` + `FooRuntime`. They do their own job and you dont _pollute_ the other class.

Also see https://github.com/symfony/ux/pull/1766#discussion_r1575257057 :D

Reply:
That would not change any class, just remove an useless one :)